langs/arturo/Dockerfile

```diff
@@ -1,24 +1,18 @@
-FROM nimlang/nim:2.2.4-alpine-slim AS builder
+FROM nimlang/nim:2.2.6-alpine-slim AS builder
 
-RUN apk add --no-cache pcre
+RUN apk add --no-cache pcre-dev
```

Comment:
Not sure how that installs faster, it's strictly a superset of packages because pcre-dev depends on pcre? https://pkgs.alpinelinux.org/package/v3.22/main/x86_64/pcre-dev

That said I'm surprised the dev headers weren't needed before, I guess because it's Nim and not C is doesn't need the header files?